### PR TITLE
artifacts.k8s.io: Create prod configuration for CloudRun

### PR DESCRIPTION
**Pull request by justinsb**
Copying the existing configuration with a few tweaks to remove sandbox etc.

### CODE REVIEW

File: infra/gcp/terraform/k8s-infra-porche-prod/network.tf [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  create_address      = false
+  create_ipv6_address = false
+  enable_ipv6         = true
+  https_redirect      = true
+  #TODO(ameukam): current the TF resource google_compute_global_address don't have
+  #the value of IP in his attribute. But it's accessible with the data source:
+  #https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/compute_global_address
+  address      = data.google_compute_global_address.default_ipv4.address
+  ipv6_address = data.google_compute_global_address.default_ipv6.address
+  managed_ssl_certificate_domains = [
```
Comment (justinsb) [inline infra/gcp/terraform/k8s-infra-porche-prod/network.tf:97]: So when we repoint DNS I think there will be about 15 minutes where we won't have a valid cert etc.  Not sure if there's any way around this...

Comment (ameukam) [reply infra/gcp/terraform/k8s-infra-porche-prod/network.tf:97]: We need to define a maintenance window of 1 hour to do the switch (happy to do the switch on a week-end).

**Comment by ameukam** [comment]
/lgtm
/hold

Remove the hold when ready.

**Comment by justinsb** [comment]
Well this won't really "do" anything until we move the DNS so I think we can apply this whenever we want to.  I don't know if we need to block cutover on having AWS artifact copying be fully automatic - we don't _have_ to, but I've been delaying completing the artifact copying until we have at least one prow job running kpromo with these mirrors.

**Comment by ameukam** [comment]
Project created, services enabled and Groups empowered as `Owner`:

```bash
$terraform apply -auto-approve -target 'google_project.project' -target 'google_project_service.project' -target 'google_project_iam_member.k8s_infra_porche_admins'
...

Releasing state lock. This may take a few moments...

Apply complete! Resources: 11 added, 0 changed, 0 destroyed.
```

**Pull request merged**